bundle-workflow/src/test_workflow/transform_MensorResults.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import json
+
+import json2table
+
+
+class JsonTransformation:
+
+    def convert_to_html(self, json_file, path_to_jsonfile):
+        with open(path_to_jsonfile + '/' + json_file) as file:
+            try:
+                infoFromJson = json.load(file)
+                testExecutionId = infoFromJson['testExecutionId']
+                build_direction = "TOP_TO_BOTTOM"
+                table_attributes = {"style": "width:100%", "border": "solid"}
+                stored = json2table.convert(infoFromJson,
+                                            build_direction=build_direction,
+                                            table_attributes=table_attributes)
+                return stored, "successful", testExecutionId
```

Comment:
Errors are swallowed and some strange structure is returned.

Reply:
So, the idea was, as the result generated might either be a json structure or error message, I thought of using try and except block for differentiating between result and error for printing that particular message into the file.

Reply:
If the result is an error, throw an exception. 

Otherwise return the result. In this case returning `successful` is unnecessary because the function will fail (exception will be raised) on error, and `testExecutionId` is already known to the caller (it's in the json output that was input to this code).